crates/flox-rust-sdk/src/models/flox_package.rs

```diff
@@ -265,10 +268,14 @@ mod tests {
     }
 
     #[test]
+    #[ignore = "In the nix sandbox the current directory is not a flake nor a repo due to file filters)"]
     fn parse_flakeref() {
         let expected = FloxPackage::Installable(Installable {
-            flakeref: runix::flake_ref::FlakeRef::Path(PathRef {
-                path: Path::new(".").to_path_buf(),
+            flakeref: runix::flake_ref::FlakeRef::GitPath(GitRef {
+                url: url::Url::from_file_path(current_dir().unwrap().ancestors().nth(2).unwrap())
```

Comment:
Is it standard to use `current_dir()` in tests? I had used `CARGO_MANIFEST_DIR` cause I thought it might be more stable

Reply:
Probably more stable yeah, lets use the manifest dir